packages/webapp/src/components/Profile/EditUser/index.jsx

```diff
@@ -57,18 +56,7 @@ export default function PureEditUser({
     ? { value: 3, label: dropDownMap[3] }
     : { value: userFarm.role_id, label: dropDownMap[userFarm.role_id] };
 
-  const getDefaultGender = () => {
-    switch (userFarm.gender) {
-      case 'MALE':
-        return genderOptions[0];
-      case 'FEMALE':
-        return genderOptions[1];
-      case 'OTHER':
-        return genderOptions[2];
-      case 'PREFER_NOT_TO_SAY':
-        return genderOptions[3];
-    }
-  };
+  const getDefaultGender = () => genderOptions.find(({ value }) => value === userFarm.gender);
```

Comment:
Any value in tossing this into the hook too?

Reply:
Ummmm I'm really not sure!

This query format is only used in two of the components and one of them had an enum which we would have to either ignore or pass... I don't think it cleans up a lot but I can push it for evaluation.

Reply:
Yah up to you! Just wondering if there was value in keeping it all in one place.